### ISSUE TITLE
Possible example GCN mistake

### ISSUE THREAD

**Issue by dylanvanw** [dmlc/dgl#4969]
## ❓ Question

I was looking at the example for GCN and found a possible mistake in the code. I opened this issue mostly to understand if the code I found is there with a specific reason or that it should not be there.
The example I am talking about: https://github.com/dmlc/dgl/blob/master/examples/pytorch/gcn/train.py

I noticed that there is a normalisation step on [line 97](https://github.com/dmlc/dgl/blob/master/examples/pytorch/gcn/train.py#L97) while the normalised values are not used downstream. Even if this was meant to show the normalisation step, it would not be calculating the normalisation step described in the [CGN paper](https://arxiv.org/abs/1609.02907). The paper considers both in and out degrees while the normalisation in the code only describes normalisation using the in degrees.
In the end, the normalised values are assigned to `g.ndata["norm"]` but these values are not used afterwards.

Having a normalisation step here is also unnecessary since the GraphConv layer that is used already takes care of the normalisation. https://docs.dgl.ai/en/0.9.x/_modules/dgl/nn/pytorch/conv/graphconv.html#GraphConv

Should that normalisation step still exist in the example? Is there a specific reason for that to be there that I am unaware of? If not, I can create a PR to remove it.

**Comment by rudongyu** [comment]
Hi, @dylanvanw. You're right. The pre-computed norm is unnecessary there. You're appreciated to create a PR to remove it. Thanks a lot!

**Comment by dylanvanw** [comment]
Hi @rudongyu! Thanks for the response. I create the PR for the changes. #4976
